src/codeu/chat/client/core/Context.java

```diff
@@ -19,13 +19,14 @@
 import java.util.Collection;
 
 import codeu.chat.common.BasicView;
+import codeu.chat.common.ServerInfo;
 import codeu.chat.common.User;
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.ConnectionSource;
 
 public final class Context {
 
-  private final BasicView view;
+  private final View view;
```

Comment:
@mbassiouny 
We changed this because at the bottom of the class, we added a method called getInfo(). It calls the view.getInfo() method. That getInfo() method isn't in the BasicView class, but it is in the View class, which inherits from BasicView. Rithu and I had both decided to go with the option of changing the variable view's type to View instead of adding the getInfo() method to BasicView. Should we add the method to BasicView instead of changing view's type?